modules/network/src/test/java/org/apache/ignite/network/DefaultMessagingServiceTest.java

```diff
@@ -102,10 +90,9 @@ class DefaultMessagingServiceTest {
     );
 
     @BeforeEach
-    void setUp() {
-        configureSender();
-        configureReceiver();
-
+    void setUp() throws InterruptedException, ExecutionException {
+        senderNetworkConfig.port().update(SENDER_PORT).get();
```

Comment:
This stuff can be simplified by providing a text config, for example:
```
@InjectConfiguration("mock.port=" + RECEIVER_PORT)
private NetworkConfiguration receiverNetworkConfig;
```

Reply:
Awesome! I tried another variant and got NPE `@InjectConfiguration("network.port=" + RECEIVER_PORT)`. Your variant works.